src/main/java/net/sf/mzmine/modules/peaklistmethods/normalization/standardcompound/StandardCompoundNormalizerTask.java

```diff
@@ -190,10 +195,7 @@ public void run() {
 
 			Feature standardPeak = standardRow.getPeak(file);
 			if (standardPeak == null) {
-			    // What to do if standard peak is not
-			    // available? (Currently this is ruled out by the
-			    // setup dialog, which shows only peaks that are
-			    // present in all samples)
+			    // What to do if standard peak is not available?
```

Comment:
I just checked the help file for the module and it says that:
"... User can select one or multiple internal standard peaks, which must be present in all raw data files."

So I guess that there is something not working with the setup dialog.